app/helpers/admin/admin_helper.rb

```diff
@@ -59,4 +59,13 @@ def render_custom_links
   def cms_title
     @page.try(:name) || "Slices CMS"
   end
+
+  # Create a link to view the current page on the live site
+  #
+  # @!visibility private
+  def link_to_view_page(page, options = {})
+    options.reverse_merge!(target: 'site', class: 'view-page')
```

Comment:
Also `options.reverse_merge!` mutates the passed-in argument, which seems a bit dangerous! The caller might re-use the `options` hash without realising it changed.